service/collector/deployment.go

```diff
@@ -14,7 +14,7 @@ import (
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/giantswarm/azure-operator/client"
-	"github.com/giantswarm/azure-operator/service/controller/v12/key"
+	"github.com/giantswarm/azure-operator/service/controller/v14/key"
```

Comment:
I updated collector references to `key` package as a preparation for upcoming flattening of `azure-operator`.